src/univalue/univalue.cpp

```diff
@@ -164,6 +164,19 @@ int UniValue::findKey(const std::string& key) const
     return -1;
 }
 
+const UniValue UniValue::findValue(const std::string& key) const
+{
+    if (typ != VOBJ)
+        return NullUniValue;
```

Comment:
I'd say throwing `std::runtime_error` would be fine,  but agreed,  separation from the trivial change should allow for merge.